src/tunacode/core/agents/agent_components/agent_config.py

```
@@ -166,62 +234,37 @@ def invalidate_agent_cache(model: str, state_manager: StateManagerProtocol) -> b
-        content = context_cache.get_context(tunacode_path)
-        version = get_or_compute_prompt_version(tunacode_path)
-
-        return content, version
-
-    except Exception as e:
-        logger.error(f"Unexpected error loading guide file: {e}")
+        return context_cache.get_context(tunacode_path), get_or_compute_prompt_version(
+            tunacode_path
+        )
+    except Exception as exc:  # noqa: BLE001
+        logger.error(f"Unexpected error loading guide file: {exc}")
```

Ruff (0.15.6)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)